optimade/validator/validator.py

```diff
@@ -423,8 +425,8 @@ def _test_must_properties(
         """
         must_props = set(
             prop
-            for prop in CONF.entry_schemas[endp]
-            if CONF.entry_schemas[endp].get(prop, {}).get("support")
+            for prop in CONF.entry_schemas.get(endp, {})
+            if CONF.entry_schemas.get(endp).get(prop, {}).get("support")
```

Comment:
I forget if `None` is returned by default or if it raises `KeyError`.
```suggestion
            if CONF.entry_schemas.get(endp, {}).get(prop, {}).get("support", None)
```

Reply:
It's `None` by default

Reply:
Great. The fallback empty dictionary should still be added however.

Reply:
I don't think the fallback is necessary as if `endp` is missing in the conditional, the iteration is over the empty dictionary above...

Reply:
Sure. But then there's also no need to use the `get` method.
And not sure how much the Python interpreter will run through. I can do a quick test though.

Reply:
Indeed, it doesn't seem to run through the `if` condition. I would still argue to either use the square brackets or the proper fallback for the `get` method.

Reply:
fair enough, done